homeassistant/components/meteoalarm/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "meteoalarm",
   "name": "MeteoAlarm",
   "documentation": "https://www.home-assistant.io/integrations/meteoalarm",
-  "requirements": ["meteoalertapi==0.1.6"],
+  "requirements": ["meteoalertapi==0.1.8"],
   "codeowners": ["@rolfberkenbosch"],
   "iot_class": "local_polling"
```

Comment:
Is it actually `local_polling`? I guess it should be `cloud_polling`

https://github.com/rolfberkenbosch/meteoalert-api/blob/69047437649b9ecedd15ae3456d988d514540856/meteoalertapi/meteoalertapi.py#L23

Reply:
you right, i have changed it. it should be cloud_polling.